tests/main/network-retry/task.yaml

```diff
@@ -20,6 +20,12 @@ restore: |
     fi
 
 execute: |
+    if [ -n "${http_proxy:-}" ] || [ -n "${https_proxy:-}" ] || [ -n "${HTTPS_PROXY:-}" ] || [ -n "${HTTPS_PROXY:-}" ]; then
```

Comment:
Breaking that line like in ` tests/main/proxy/task.yaml` would be nice, but probably not worth another spread run.